appinventor/appengine/war/static/css/Ya.css

```diff
@@ -3145,15 +3256,7 @@ div.dropdiv p {
   margin-right: auto;
 }
 
-.download-barcode:before {
-    content: "";
-    background-color: #888;
-    position: absolute;
-    width: 1px;
-    height: 160px;
-    top: 40px;
-    left: 50%;
-    display: block;
```

Comment:
What is the justification for removing this?

Reply:
Perhaps this would be better as a call for help. I've found the before syntax for css to be very difficult to work with in UIBinder. I eventually rage quit, since I don't see that this vertical line brings much to the dialog anyway.